library/src/scripts/theming/ThemeProvider.tsx

```diff
@@ -92,7 +92,7 @@ export const ThemeProvider: React.FC<IProps> = (props: IProps) => {
 
     return (
         <>
-            <Backgrounds />
+            <Backgrounds isHomePage={true} />
```

Comment:
@charrondev I'm not sure how I can detect the home page here. What do you suggest?